GitUI/FormSignatureInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using GitCommands.Utils;
+
+namespace GitUI
+{
+    public partial class FormSignatureInfo : GitExtensionsForm
+    {
+        public FormSignatureInfo()
```

Comment:
the window should also remember the last size
to retain the size, this has to pass `true` to the base class

Reply:
ok